[Platform] Derive VertexAI API host from location for regional and data-residency endpoints (#2284)
symfony/ai: src/platform/src/Bridge/VertexAi/CHANGELOG.md

0.11
----

>  * Derive the API host from the configured `location` so regional (e.g. `europe-west1`) and data-residency (`eu`, `us`) endpoints are used; `null`/`global` keep the existing `aiplatform.googleapis.com` host

Reviewer: let's keep it short:

```suggestion
 * Derive the API host from the configured `location` to support regional and and data-residency endpoints
```
